src/components/MapView/DateSelector/utils.ts

```diff
@@ -15,6 +15,38 @@ export const months = [
   'Dec',
 ];
 
+/**
+ * Return the closest date from a given list of available dates
+ * @param date
+ * @param availableDates
+ * @return date as momentjs object
+ */
+export function findClosestDate(
+  date: number,
+  availableDates: ReturnType<Date['getTime']>[],
+) {
+  const dateToCheck = moment(date);
+  let closest: any;
```

Comment:
Can we avoid the use of any? Maybe if you use `array.reduce` we could ensure strong typing, and also remove the eslint warning `fp/no-mutation`

Reply:
@Ry-DS thanks for the input. It is a good idea and changes have been made. Could you please re-check again?